packages/react/src/utils.ts

```diff
@@ -73,4 +74,35 @@ const useSize = (target: React.RefObject<HTMLDivElement>) => {
   return size;
 };
 
-export { mergeProps, LKComponentAttributes, useObservableState, cloneSingleChild, useSize };
+export type TokenizeGrammar = { [type: string]: RegExp };
+
+export function tokenize(input: string, grammar: TokenizeGrammar) {
+  const matches = Object.entries(grammar)
```

Comment:
Not important: `type TokenizeGrammer` and `function tokenize` could probably be moved to `/packages/core/` as they are not React specific and could be used in other implementations in the future.

Reply:
ah, sorry paul, that's my fault. @Ocupe is right, I misdirected you to `utils` in the react package for everything.